src/common/utils/url.ts

```diff
@@ -212,7 +217,7 @@ function isCustomLoginURL(url, server, teams) {
         return false;
     }
     const urlPath = parsedURL.pathname;
-    if ((subpath !== '' || subpath !== '/') && urlPath.startsWith(subpath)) {
+    if (urlPath.startsWith(subpath)) {
```

Comment:
The statement `subpath !== '' || subpath !== '/'` will always evaluate to `true`. A string will always be not equal to one of those conditions. If you refactor using DeMorgan's it's a bit clearer:
`subpath !== '' || subpath !== '/' -> !(subpath === '' && subpath === '/')`

Reply:
oh! that was supposed to be an `&&` seems like I made a mistake when I added that.

The reason I added it is because a subpath will usually be like '/subpath', and URL.pathname gives you '/' as a minimum (which will always be part of the subpath). I don't remember why did I add the empty string one

Reply:
Okay that makes more sense, I'll fix it here.